extensions/functions_arithmetic.yaml

```diff
@@ -274,6 +274,28 @@ scalar_functions:
             required: false
           - value: fp64
         return: fp64
+  -
+    name: "exp"
+    description: "The mathematical constant e, raised to the power of the value."
+    impls:
+      - args:
+          - name: rounding
+            options: [ TIE_TO_EVEN, TIE_AWAY_FROM_ZERO, TRUNCATE, CEILING, FLOOR ]
+            required: false
+          - name: on_domain_error
+            options: [ NAN, ERROR ]
+            required: false
```

Comment:
Isn't e^x is defined for all real numbers?

Reply:
Oops, fixed.